### DISCUSSION TITLE
Plot current position and history position

### DISCUSSION THREAD

**Discussion by bohlinz** [foxglove/community#1056] [Q&A]
Hi,

I have a signal for the positions in [x,y] and a plot layout is used to draw the history position.
The config is as follows.

![image](https://github.com/user-attachments/assets/4c501bca-5e82-49d0-9d29-1c5142b7a1bb)

But I also want to plot the current position. It seems not just simply add another y_m channel and plot current.

Any suggestion? 
Thanks in advance.

**Comment by defunctzombie** [comment]
Does the current position come from the same topic `/display/state_estimation.y_m`?

**Reply by bohlinz** [reply]
yes. I want to use the same topic but one plot for history and a point on the current position at the same figure.

**Reply by indirectlylit** [reply]
thanks for the suggestion!

we're tracking this internally under FG-9681
